.authors.yml

```diff
@@ -1,6 +1,6 @@
 - name: Michael Sarahan
   email: msarahan@continuum.io
-  num_commits: 1330
+  num_commits: 665
```

Comment:
This is going down by using the current dev version of rever which implements counting correctly: https://github.com/regro/rever/pull/266